recipes/cWB/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -ex
+
+mkdir -pv build
+pushd build
+
+cmake \
+  -DCMAKE_BUILD_TYPE:STRING=Release \
+  -DCMAKE_INSTALL_LIBDIR:PATH="lib" \
+  -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} \
+  ${SRC_DIR}
+
+# build
+cmake --build . --parallel ${CPU_COUNT} --verbose > build.log
+
+# install
+cmake --build . --parallel ${CPU_COUNT} --verbose --target install
+
+# install activate/deactivate scripts
+for action in activate deactivate; do
+	mkdir -p ${PREFIX}/etc/conda/${action}.d
+        _target="${PREFIX}/etc/conda/${action}.d/activate-${PKG_NAME}.sh"
```

Comment:
The deactivate script shouldn't be called `activate` (you probably copied this from me, which I think I have fixed in most places now):

```suggestion
        _target="${PREFIX}/etc/conda/${action}.d/${action}-${PKG_NAME}.sh"
```